cmd/helm/list_test.go

```diff
@@ -56,9 +56,7 @@ func TestListCmd(t *testing.T) {
 			rels: tt.resp,
 		}
 		cmd := newListCmd(c, &buf)
-		for flag, value := range tt.flags {
-			cmd.Flags().Set(flag, value)
-		}
+		setFlags(cmd, tt.flags)
```

Comment:
I don't understand what you are suggesting I do here. Simply putting `args: []string{"--long"}` doesn't work.